pkg/storage/mock/storer.go

```diff
@@ -60,7 +63,7 @@ func (m *MockStorer) Put(ctx context.Context, mode storage.ModePut, chs ...swarm
 		}
 		m.store[ch.Address().String()] = ch.Data()
 	}
-	return nil, nil
+	return []bool{true}, nil
```

Comment:
this is incorrect. the size of this slice should be the same length as `len(chs)`

Reply:
Yes, this should be corrected.